src/deploydocs.jl

```diff
@@ -357,7 +365,7 @@ function git_push(
         end
 
         # Add, commit, and push the docs to the remote.
-        run(`$(git()) add -A .`)
+        run(`$(git()) add -A -- ':!.documenter-identity-file.tmp' ':!**/.documenter-identity-file.tmp'`)
```

Comment:
@fredrikekre Does this look right to you? `:!.documenter-identity-file.tmp'` seemed to only exclude the top-level file.

Reply:
Looks like that works, but I wonder why the first isn't sufficient. In a standard `.gitignore` file a line with `.documenter-identity-file.tmp` would exclude all files with that name.